packages/python-packages/apiview-copilot/evals/run.py

```diff
@@ -74,10 +74,10 @@ def __call__(self, *, response: str, query: str, language: str, output: str, **k
         return review_eval
 
 
-def review_apiview(query: str, language: str):
+def review_apiview(query: str, language: str, model: Literal["gpt-4o-mini", "o3-mini"]="o3-mini") -> dict[str, Any]:
```

Comment:
Any chance we could set the default model in APIViewReview? This function is called with the parameters from the JSONL template and doesn't take model as a parameter currently. We could also choose to update the template, but I think it makes more sense to control the model we're using /defaulting to outside the evals code.

Reply:
I hard-coded the model selection in the body of the `review_apiview` method.